src/proto/play.proto

```diff
@@ -4,29 +4,23 @@ package TbotsProto;
 
 enum PlayName
 {
-    UseAiSelection                     = 1;
-    ShootOrChipPlay                    = 2;
-    KickoffFriendlyPlay                = 3;
-    OffensePlay                        = 4;
-    CornerKickPlay                     = 5;
-    ShootOrPassPlay                    = 6;
-    BallPlacementPlay                  = 7;
-    HaltPlay                           = 8;
-    EnemyFreeKickPlay                  = 9;
-    ExamplePlay                        = 10;
-    DefensePlay                        = 11;
-    EnemyBallPlacementPlay             = 12;
-    PenaltyKickEnemyPlay               = 13;
-    PenaltyKickPlay                    = 14;
-    FreeKickPlay                       = 15;
-    CreaseDefensePlay                  = 16;
-    StopPlay                           = 17;
-    KickoffEnemyPlay                   = 18;
-    AssignedTacticsPlay                = 19;
-    ScoringWithStaticDefendersPlay     = 20;
-    DribblingParcourPlay               = 21;
-    ScoringFromContestedPossessionPlay = 22;
-    PassEndurancePlay                  = 23;
+    reserved 2, 5, 6, 16;
+
+    UseAiSelection         = 1;
```

Comment:
I imagine you're ordering this for displaying it in Thunderscope but it seems a little confusing. How did you order this list?

Reply:
Yeah i reordered it for thunderscope. I just put the friendly/enemy versions of each play together

Reply:
Ah I see now